app/models/merchant.rb

```diff
@@ -0,0 +1,5 @@
+class Merchant < ApplicationRecord
+
+  validates_presence_of :name, :address, :city, :state, :zip
+
+end
```

Comment:
This merchant would eventually have a has_many relationship with items since a merchant has many items and an item belongs to a merchant.